src/DotFlyer.Api/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+USER app
+WORKDIR /app
+EXPOSE 8080
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+COPY ["DotFlyer.Api/DotFlyer.Api.csproj", "DotFlyer.Api/"]
+RUN dotnet restore "./DotFlyer.Api/DotFlyer.Api.csproj"
+COPY . .
+WORKDIR "/src/DotFlyer.Api"
+RUN dotnet build "./DotFlyer.Api.csproj" -c $BUILD_CONFIGURATION -o /app/build
+
+FROM build AS publish
```

Comment:
I should/could we just use the published artifact from the pipeline here instead of publishing again?